httpclient/client.go

```diff
@@ -0,0 +1,42 @@
+package httpclient
+
+import (
+	"fmt"
+	"net/http"
+
+	transactionidutils "github.com/Financial-Times/transactionid-utils-go"
+)
+
+type httpClient interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
+type ServiceClient struct {
+	Client      httpClient
+	ServiceCode string
+	Version     string
+	Runbook     string
+}
+
+func (c *ServiceClient) Do(req *http.Request) (*http.Response, error) {
+	headers := req.Header.Values("User-Agent")
+	// don't override "User-Agent" if it's already set
+	if len(headers) == 0 {
```

Comment:
`headers` does not reflect the actual value of the variable, which is the values of the user agent header.